playlists/word.yaml

```diff
@@ -191,6 +191,15 @@
   group: Scenarios
```

Comment:
Instead of Scenarios, consider putting this under the "Content Controls" category.

Reply:
Actually, it's in the Preview APIs group. It'll be put under "Content Controls" if/when the onDeleted event GAs